chapter_6/source_code/fingerprint/fingerprint_process/fingerprint_process.cpp

```diff
@@ -69,11 +69,11 @@ void thinning(Mat& im)
     im *= 255;
 }
 
-int main( int argc, const char** argv )
```

Comment:
I agree that this is not been used so that it could be removed. This is just how I set up my default project layouts.